CHANGES/6494.bugfix

```diff
@@ -0,0 +1 @@
+Support credentials in URL with empty user, e.g. http://:password@host
```

Comment:
Plz use the past tense here.

Reply:
How about:
"URLs providing credentials with zero-length user are now supported (http://:password@host)" ?